lib/Cache/Core/CoreHandler.php

```diff
@@ -270,8 +276,17 @@ public function load($key)
 
         if ($item->isHit()) {
             $data = $item->get();
+
             $data = unserialize($data);
 
+            $context = [
+                'source' => __METHOD__,
+                'conversion' => 'unmarshal'
+            ];
+            $copier = Service::getDeepCopyInstance($data, $context);
```

Comment:
Can we avoid this somehow? 
Doing a deep copy on load is slowing down the retrieval of data from the cache, so if possible we shouldn't do this. If necessary, we'd have to do some extensive performance testing so see how big the impact of this is.

Reply:
Do we really need that? This seems to be unnecessary as the dump state is anyway set to `false` before serialization, see: https://github.com/pimcore/pimcore/pull/7051/files#diff-903c6798cb9358159948a1eca1bb31fdR594 so on unserializing it it should be still `false` ... 
@weisswurstkanone can you please check - thanks!